docs/en/tutorials/getting-started.rst

```diff
@@ -717,8 +717,8 @@ the bi-directional reference:
     {
         // ... (previous code)
 
-        private $reportedBugs = null;
-        private $assignedBugs = null;
+        protected $reportedBugs = null;
```

Comment:
I'd rather have everything as `private` around the code, but if you think that `protected` is better for consistency (for now), I can merge the PR as-is.

Should generally strive for `private` in future.